man/ronin-archive.1.md

```diff
@@ -6,40 +6,41 @@ ronin-archive - Archive the files.
 
 ## SYNOPSIS
 
-`ronin archive` [*options*] [`-t` \| `--tar`] [`-z` \| `--zip`]  {`-o`,`--output` *FILENAME* } [*FILE* ...]
+`ronin archive` [*options*] [`-t` \| `--tar`] [`-z` \| `--zip`]  {`-o`,`--output` *s/FILENAME/PATH/* } [*FILE* ...]
 
 ## DESCRIPTION
 
 Archive the files.
 
 ## ARGUMENTS
 
-[*FILE* ...]
-: Files to archive.
+*FILE*
+: A file to add to the output archive.
 
-## OPTIONS
+*PATH*
+: A path to the output file.
```

Comment:
Put `*PATH*` directly after `--output`, but on the same line.

    `-o`, `--output` *PATH*